src/set_theory/ordinal_arithmetic.lean

```diff
@@ -2019,6 +2028,20 @@ le_antisymm
     (λ b hb, le_of_lt (opow_lt_omega h hb)))
   (le_opow_self _ a1)
 
+protected theorem is_normal.omega {f : ordinal.{u} → ordinal.{u}} (hf : is_normal f) :
```

Comment:
```suggestion
protected theorem is_normal.apply_omega {f : ordinal.{u} → ordinal.{u}} (hf : is_normal f) :
```
maybe?

Reply:
Sure, I didn't really like the original name.